src/frontend/Sider/Behandling/Vilkårvurdering/EndreDelvilkår.tsx

```diff
@@ -137,6 +148,26 @@ const EndreDelvilkår: FC<{
         settFeilmeldinger({ ...feilmeldinger, [regelId]: undefined });
     };
 
+    if (!erIRedigeringsmodus) {
```

Comment:
For å ikke få for mye i denne komponenten hadde nesten dette kunnet vært en nivå opp

Reply:
Enig i den! Tar en titt på forslaget til Sara

Reply:
Fiksa!

Reply:
Redigeringsmoduss er tatt ut avkomponenten